tests/test_metadata.py

```diff
@@ -16,33 +16,47 @@ class TestEnvelope(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         cls.signature_dict = {
-            "keyid": "11fa391a0ed7a447cbfeb4b2667e286fc248f64d5e6d0eeed2e5e23f97f9f714",
-            "sig": "30460221009342e4566528fcecf6a7a5d53ebacdb1df151e242f55f8775883469cb01dbc6602210086b426cc826709acfa2c3f9214610cb0a832db94bbd266fd7c5939a48064a851",
+            "keyid": "11fa391a0ed7a447",
+            "sig": "30460221009342e4566528fcecf6a7a5",
+        }
+        cls.gpg_signature_dict = {
+            "keyid": "f4f90403af58eef6",
+            "signature": "c39f86e70e12e70e11d87eb7e3ab7d3b",
```

Comment:
Is this an actual signature or just dummy data?

Reply:
It's a dummy data not a real signature, because these dicts are only used to test serialization and deserialization of the DSSE Envelope.